CorsixTH/Lua/humanoid_actions/seek_room.lua

```diff
@@ -188,7 +198,7 @@ local function action_seek_room_start(action, humanoid)
     humanoid:finishAction()
     return
   end
-  -- Tries to find the room, if it is a diagnosis room, try to find any room in the diagnosis list.
+  -- Try to find the appropriate room for the patient
   if not humanoid.diagnosed or action.room_type == "research" then
     local room = action_seek_room_find_room(action, humanoid)
```

Comment:
Think the `local` can be dropped from this line now? Just want someone to confirm.

Reply:
Thanks github actions,for confirming

Reply:
(I actually meant line 68)